### ISSUE TITLE
'machine code' bytes are printed in big-endian order

### ISSUE THREAD

**Issue by bjacob** [wolfpld/tracy#136]
Tracy is showing this machine code and disassembly:

![Untitled 3](https://user-images.githubusercontent.com/79535/100407014-442afd00-3035-11eb-9b08-467d719284fc.jpg)

According to the ARM architecture reference manual, C7.2.176 LDR (immediate, SIMD&FP), section "Unsigned offset", the encoding as a 32-bit word is:

![image](https://user-images.githubusercontent.com/79535/100407127-9409c400-3035-11eb-91fb-c130bb1a46cb.png)
![image](https://user-images.githubusercontent.com/79535/100407166-ab48b180-3035-11eb-8914-e782493b3f34.png)

We can see here that it is the most significant bits of this 32bit word that are fixed (independent of operands) and that the most significant 8 bits are `00111101 = 0x3D`.

In the Tracy view, the byte value '3D' is shown as the last of the 4 bytes for this instruction, so it's big-endian.

By contrast, `objdump -d` is showing this in little-endian order:

```
    2534:       3dc002c2        ldr     q2, [x22]
    2538:       3dc002e1        ldr     q1, [x23]
    253c:       3dc00300        ldr     q0, [x24]
    2540:       3dc00323        ldr     q3, [x25]
```

This is more convenient for me because if I need to put this in a uint32, I can just copy and paste it and prefix it with `0x`.

**Comment by wolfpld** [comment]
I believe you have little- and big-endian the wrong way around here.

Should be fixed by 3495a5a67.

**Comment by bjacob** [comment]
Thanks for the correction, I did have it the wrong way around. Thanks for the fix anyway!
